bin/glow-monitor/main.go

```diff
@@ -4,23 +4,44 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
+	"path/filepath"
 	"syscall"
 
 	"github.com/glowlabs-org/gca-backend/client"
 )
 
 func main() {
 	// Create a new client, using the current directory as the basedir.
-	c, err := client.NewClient("/opt/glow-monitor/")
+	baseDir := "/opt/glow-monitor/"
+	c, err := client.NewClient(baseDir)
 	if err != nil {
 		fmt.Println("unable to create client: ", err)
 		return
 	}
 
 	// Wait for a shutdown signal from the OS.
 	sigChan := make(chan os.Signal, 1)
-	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
-	<-sigChan
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGUSR2)
```

Comment:
I know it's not your code, but it might be worth increasing the buffer size on sigChan